cdap-ui/app/cdap/components/Reports/ReportsDetail/Runs/Runs.scss

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import '../../../../styles/variables.scss';
+
+$header-bg-color: $grey-08;
+$border-color: $grey-03;
+
+.reports-runs-container {
+  // 40px height of action section
+  // 182px height of summary section plus border
+  height: calc(100% - 40px - 182px);
+
+  .grid-wrapper {
+    height: 100%;
+    min-height: 300px;
+  }
+
+  .grid.grid-container {
+    max-height: 100%;
+
+    .grid-header,
+    .grid-body {
+      padding: 0 35px;
+
+      .grid-row > * {
+        padding: 5px 10px;
```

Comment:
***Not related to this PR***
So the idea behind us using a common grid for tables is to have a consistent look and feel for tables across cdap ui. If we change the padding and background font size then I think the purpose of using a common table across UI is going away. @leacd Is there a reason why we do that? Can we have the same header bg color, font size, padding and margin across all the tables?

I myself have been doing this and now I realize that it is not giving a consistent look across UI. May be we should sync up on the table styles and decide what should be common across everywhere and what should be configurable (changeable).

Reply:
I have been trying to re-use the same designs for tables.  I can think of 3 different grids/tables I have been using. I don't think we should be rigid and box us in with one specific design.  As we are starting to use tables a lot more, I don't think that 1 size fits all can help us.
I'll be happy to chat more about it.